### PR TITLE
NAD UI and Fund transfer UI fixed

### PR DESCRIPTION
**Pull request by hiteshdigit101**

### CODE REVIEW

File: src/components/adb-nad-account-link/index.tsx [hunk 1]
```diff
@@ -180,17 +180,15 @@ const NADAccountLinkComponent = (prop: NADAccountLinkComponentProps) => {
...
     if (isSuccess.success) {
       const arr = [...beneficiaryTypes];
       arr[proxy.index].status = "Active";
       setBeneficiaryTypes(arr);
-      setSuccessModalTitle(
-        "Your DuitNow ID is successfully linked with your {ADB} account"
-      );
+      setSuccessModalTitle("DuitNow ID activated!");
       setSuccessModalSubTitle(
         "Your DuitNow ID has been successfully reactivated"
```
Comment (sharmal33) [inline src/components/adb-nad-account-link/index.tsx:193]: move this to translations file

File: src/components/transfer-details-component/index.tsx [hunk 2]
```diff
@@ -242,18 +246,21 @@ const TransferDetailsComponent = ({
                       errorTextStyle: {
                         color: "#1b1b1b",
```
Comment (sharmal33) [inline src/components/transfer-details-component/index.tsx:247]: use color palette

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-nad-account-link/index.tsx (hunk 1 above)
Comment (hiteshdigit101) [reply src/components/adb-nad-account-link/index.tsx]: moved to translation file

File: src/components/transfer-details-component/index.tsx (hunk 2 above)
Comment (hiteshdigit101) [reply src/components/transfer-details-component/index.tsx]: changed from static color to theme component colors

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**